composer.json

```diff
@@ -14,21 +14,21 @@
     ],
 
     "require": {
-        "php": "^7.2 || ^8.0",
+        "php": "8.1.* || 8.2.* || 8.3.* || 8.4.* ",
         "ext-mbstring": "*",
         "behat/gherkin": "^4.10.0",
-        "behat/transliterator": "^1.2",
-        "symfony/console": "^4.4 || ^5.0 || ^6.0 || ^7.0",
-        "symfony/config": "^4.4 || ^5.0 || ^6.0 || ^7.0",
-        "symfony/dependency-injection": "^4.4 || ^5.0 || ^6.0 || ^7.0",
-        "symfony/event-dispatcher": "^4.4 || ^5.0 || ^6.0 || ^7.0",
-        "symfony/translation": "^4.4 || ^5.0 || ^6.0 || ^7.0",
-        "symfony/yaml": "^4.4 || ^5.0 || ^6.0 || ^7.0",
+        "behat/transliterator": "^1.5",
+        "symfony/console": "^5.4 || ^6.4 || ^7.0",
+        "symfony/config": "^5.4 || ^6.4 || ^7.0",
+        "symfony/dependency-injection": "^5.4 || ^6.4 || ^7.0",
+        "symfony/event-dispatcher": "^5.4 || ^6.4 || ^7.0",
+        "symfony/translation": "^5.4 || ^6.4 || ^7.0",
+        "symfony/yaml": "^5.4 || ^6.4 || ^7.0",
         "psr/container": "^1.0 || ^2.0"
     },
 
     "require-dev": {
-        "symfony/process": "^4.4 || ^5.0 || ^6.0 || ^7.0",
+        "symfony/process": "^5.4 || ^6.4 || ^7.0",
         "phpunit/phpunit": "^8.5 || ^9.0",
```

Comment:
We can simplify this. We don't need to support phpunit 8 in our testsuite anymore if we don't support older PHP versions anymore.

Reply:
@stof I have that locally already